policies/README.md

```diff
@@ -0,0 +1,64 @@
+# OPA policies
+Policies are rules that can be applied when any of your cluster object is _createdn_, _updated_, _patched_ and _deleted_.
+Policies are always executed _before_ the action takes place and their goal is to allow or prevent the requested modification.
+
+This solution is very fast in term of required computing time thanks to the efficienty of the OPA gatekeeper.
+
+
+## How to use policies
+To use OPA gatekeeper you need to:
+- Deploy a Release using Prebuilt Image. See [link](../infrastructure/policy-enforcement/README.md).
+- Create your own constraint, template and eventually config-sync. See [examples](../infrastructure/policy-enforcement/examples).
+
+Now you can test your policies on your cluster objects.
+
+
+## How to use the Gatekeeper
+
+### Template
+In the Template you should define your own constraint CRD and associate it to your rego policies, such as in this [**template.yaml**](../infrastructure/policy-enforcement/examples/template.yaml) example.
```

Comment:
"rego" was never mentioned before. Reader has no idea what this is.
Possible rephrasing: ... to your rego (i.e., the language used by the OPA gatekeeper) policies....